.drone.yml

```diff
@@ -178,15 +178,6 @@ pipeline:
     when:
       event: [ push, tag ]
 
-  docker:
-    image: plugins/docker:17.05
-    pull: true
-    secrets: [ docker_username, docker_password ]
-    repo: gitea/gitea
-    tags: [ '${DRONE_TAG##v}' ]
-    when:
-      event: [ tag ]
-
   docker:
     image: plugins/docker:17.05
```

Comment:
Seeing as the announcement was Oct '17 (17.10) should we have to update this image tag as well?